lib/common/containers/FeedLabel.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import { deleteLabel } from '../../manager/actions/labels'
+import FeedLabel from '../components/FeedLabel'
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    user: state.user
+  }
+}
+
+const mapDispatchToProps = {
+  deleteLabel
+}
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(FeedLabel)
```

Comment:
In practice with newer components, I believe we've abandoned the separation of components into connected containers and the components, so it's fine to combine this in just the component file.

Reply:
Ah thank you! Refactored in a1bc3e9